compiler/test/compilable/b20243.d

```diff
@@ -0,0 +1,12 @@
+module b20243;
+
+void main()
```

Comment:
compilable test function should not be called main

Reply:
Removed `void main() {` and `}`.